sebs/aws/config.py

```diff
@@ -199,10 +226,43 @@ def http_api(
             self.logging.info(f"Using cached HTTP API {api_name}")
         return http_api
 
+    def check_ecr_repository_exists(self, ecr_client, repository_name):
+        try:
+            ecr_client.describe_repositories(repositoryNames=[repository_name])
+            return True
+        except ecr_client.exceptions.RepositoryNotFoundException:
+            return False
+        except Exception as e:
+            self.logging.error(f"Error checking repository: {e}")
+            raise e
+
+    def create_ecr_repository(self, boto3_session: boto3.session.Session):
+        ecr_client = boto3_session.client(service_name="ecr", region_name=cast(str, self._region))
+
+        if not self._container_repository:
+            random_name = str(uuid.uuid4())[0:16]
+            repository_name = "sebs-benchmark-{}".format(random_name)
```

Comment:
Can we use here the resources id instead of a random name? When all resources have the same name, it's easier to identify everything we create.

https://github.com/spcl/serverless-benchmarks/blob/0b8d77777f0f47fa6dc10fc6247c4b539fcf772a/sebs/faas/config.py#L76

Reply:
yes, that will be useful for creating resources using the resource name. 
I have changed it.